spec/puppet_fixtures/fixtures_spec.rb

```diff
@@ -6,9 +8,6 @@
 
   let(:source_dir) { File.join('spec/fixtures/missing') }
 
-  describe '#clean' do
```

Comment:
Perhaps instead of removing it add a `TODO`?

Reply:
done